crates/holochain_keystore/src/lib.rs

```diff
@@ -44,27 +50,63 @@ pub mod lair_keystore;
 
 pub mod paths;
 
+#[cfg(feature = "test_utils")]
```

Comment:
Should be able to put this behind test_utils but it's used by the conductor with the `danger-lair-test-keystore` option... Why is that a thing?